doc/designs/ddd/Makefile

```diff
@@ -1,26 +1,36 @@
 #
-# To run the demos when linked with a shared library (default):
+# To run the demos when linked with a shared library (default) ensure that
+# libcrypto and libssl are on the library path. For example to run the
+# ddd-01-conn-blocking-tls demo:
 #
-#    LD_LIBRARY_PATH=../.. make test
+#    LD_LIBRARY_PATH=../../.. ./ddd-01-conn-blocking-tls
```

Comment:
I wonder if it might be better to use `util/wrap.pl` for setting stuff up?  Just a thought.

Reply:
I'm not sure.

I used `LD_LIBRARY_PATH` to be consistent with the other QUIC demos (under `demos/guide`).